DOCS-12593 adding antivirus statement to BYOK (#535)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/limitations-self.adoc

| Object store persistence | To manage data persistence across Mule application replicas and restarts, Anypoint Runtime Fabric uses Persistence Gateway. See xref:persistence-gateway.adoc[Persistence Gateway].
| ServiceType limitations | By default, services are deployed with the ClusterIP ServiceType. Depending on the architecture, these services may not be compatible with ingress controllers that require the NodePort ServiceType.
|===

== How Antivirus and DPI Software Impacts Runtime Fabric Functionality

The Anypoint Runtime Fabric appliance has specific hardware and OS requirements. MuleSoft Support provides Support SLA and Severity Levels based on our validation and certification of the requirements specified in the xref:install-prereqs.adoc[installation prerequisites].

> A small number of third-party software that requires root access to the host, which includes, but is not limited to, antivirus and DPI software, has been found to interfere with the behavior and requirements of the Runtime Fabric appliance. In many support cases where MuleSoft Support has detected antivirus software, they have seen issues such as port blocking, node-to-node communication blocking, Docker default bridge deletion, and filesystem access issues that interfere with Runtime Fabric appliance installation, upgrade, and normal operations.

Reviewer: A small number ... require ... have been found 

Or consider rewording to "Some third-party software requires ..."

> A small number of third-party software that requires root access to the host, which includes, but is not limited to, antivirus and DPI software, has been found to interfere with the behavior and requirements of the Runtime Fabric appliance. In many support cases where MuleSoft Support has detected antivirus software, they have seen issues such as port blocking, node-to-node communication blocking, Docker default bridge deletion, and filesystem access issues that interfere with Runtime Fabric appliance installation, upgrade, and normal operations.

Author: ```suggestion
Some third-party software requires root access to the host, which includes, but is not limited to, antivirus and DPI software, has been found to interfere with the behavior and requirements of the Runtime Fabric appliance. In many support cases where MuleSoft Support has detected antivirus software, they have seen issues such as port blocking, node-to-node communication blocking, Docker default bridge deletion, and filesystem access issues that interfere with Runtime Fabric appliance installation, upgrade, and normal operations.
```
